frontend/src/concepts/pipelines/EnsureCompatiblePipelineServer.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {
+  EmptyState,
+  EmptyStateVariant,
+  Bullseye,
+  Spinner,
+  EmptyStateHeader,
+} from '@patternfly/react-core';
+import NoPipelineServer from './NoPipelineServer';
+import { usePipelinesAPI } from './context';
+
+type EnsureCompatiblePipelineServerProps = {
+  children: React.ReactNode;
+};
+
+const EnsureCompatiblePipelineServer: React.FC<EnsureCompatiblePipelineServerProps> = ({
+  children,
+}) => {
+  const { pipelinesServer } = usePipelinesAPI();
+
+  if (pipelinesServer.initializing) {
+    return (
+      <Bullseye>
+        <Spinner />
+      </Bullseye>
+    );
+  }
+
+  if (!pipelinesServer.installed) {
+    return <NoPipelineServer variant="secondary" />;
+  }
+
+  if (!pipelinesServer.compatible) {
+    return (
+      <Bullseye data-testid="incompatible-pipelines-server">
+        <EmptyState variant={EmptyStateVariant.lg}>
+          <EmptyStateHeader>
+            The data science pipeline version installed is incompatible. Delete the DSPA and
+            recreate
```

Comment:
```suggestion
            This data science project contains an unsupported version of Data Science Pipelines.
            Delete the pipeline server and recreate it.
```

Reply:
Probably want docs to give the microcopy here long term, but I at least wanted to remove mention of "DSPA" because users won't know what that is

Reply:
Yeah this UX will be iterated on, so this is more of a placeholder. I can commit your suggestion tho.